manifest.yml

```diff
@@ -1,12 +1,13 @@
 ---
 applications:
-- name: forms-api-((PAAS_ENVIRONMENT))
-  memory: 256M
-  command: bundle exec rackup -o 0.0.0.0 -p $PORT
-  services:
-    - forms-api-((PAAS_ENVIRONMENT))-db
-    - forms-api-((PAAS_ENVIRONMENT))-splunk
-  env:
-    RACK_ENV: production
-    SENTRY_DSN: ((SENTRY_DSN))
-    API_KEY: ((API_KEY))
+  - name: forms-api-((PAAS_ENVIRONMENT))
+    memory: 256M
+    command: bundle exec rackup -o 0.0.0.0 -p $PORT
+    services:
+      - forms-api-((PAAS_ENVIRONMENT))-db
+      - forms-api-((PAAS_ENVIRONMENT))-splunk
+    env:
+      RACK_ENV: production
+      SENTRY_DSN: ((SENTRY_DSN))
+      API_KEY: ((API_KEY))
+      PAAS_ENVIRONMENT: ((PAAS_ENVIRONMENT))
```

Comment:
is this indentation correct?

Reply:
Not sure - VS Code did it automatically based on the editorconfig file. It's usually pretty good at not changing the  meaning when it does these things, and I don't _think_ Yaml cares about the difference between:
```
a
- b
```

and

```
a
  - b
```

but I'm happy to change it back if it breaks the deploy

Reply:
I've just merged something similar in the admin and it looks like the deploy ran OK - https://github.com/alphagov/forms-runner/actions/runs/2876513134

Reply:
Going to resolve this for now, we can revisit if we spot any issues (or if other people's editors start changing it back)